src/frontend/src/common/queries/phyloRuns.ts

```diff
@@ -43,14 +44,18 @@ export const USE_PHYLO_RUN_INFO = {
  * custom hook to automatically expire tree info when needed
  * such as when trees are deleted
  */
-export function usePhyloRunInfo(): UseQueryResult<PhyloRunResponse, unknown> {
-  return useQuery([USE_PHYLO_RUN_INFO], fetchPhyloRuns, {
+export function usePhyloRunInfo(
```

Comment:
instead of having every single consumer pass in the pathogen, you can read it from redux directly. See an example of how to read from the redux store without a hook at `api/index.tsx:28-34`

Reply:
Nice, thank you!

Reply:
@mmmmmmaya Making this change made switching between pathogens noticeably slower.  Any idea why?

Reply:
Seems like it's because `useSampleInfo` isn't called again when the new tab loads.  The components are no longer watching for changes to the current pathogen.  I know putting the hook inside of useEffect is not good, but I essentially want something that will watch for when the current pathogen changes and then get new data when it does.

Reply:
I think you should be able to use redux middleware to run additional code (like expire an fe cache) whenever you dispatch a setCurrentPathogen redux action. You can see examples of existing middleware in middleware.ts